### PR TITLE
update upstream to v0.15.0 and replace C-Lightning-REST with CLNRest

### PR DESCRIPTION
**Pull request by Dominion5254**

### CODE REVIEW

**Review by kn0wmad — APPROVED** [review]

File: configurator/src/main.rs [hunk 1]
```diff
@@ -212,15 +215,16 @@ pub enum Property {
...
     match (&s9_node.typ, &s9_node.connection_settings) {
         (NodeType::Lnd, S9NodeConnectionSettings::Internal) => {
-            Ok((String::from("lnd.embassy"), 8080, PathBuf::from("/mnt/lnd")))
+            Ok((String::from("lnd.embassy"), 8080, Some(PathBuf::from("/mnt/lnd")), None))
         }
         (NodeType::CLightning, S9NodeConnectionSettings::Internal) => Ok((
             String::from("c-lightning.embassy"),
-            3001,
-            PathBuf::from("/mnt/c-lightning"),
+            3010,
```
Comment (dr-bonez) [inline configurator/src/main.rs:225]: Port changed?

File: docker_entrypoint.sh [hunk 2]
```diff
@@ -37,10 +37,10 @@ if [[ ! -z $result ]]
     exit 0
   fi
 
-  while ! test -f /mnt/c-lightning/access.macaroon
+  while ! test -f /mnt/c-lightning/clnrest_rune
   do
-    echo "Waiting for c-Lightning-REST access macaroon to be generated..."
-    sleep 1
+    echo "Waiting for clnrest_rune to be generated..."
+    sleep 30
```
Comment (dr-bonez) [inline docker_entrypoint.sh:43]: Why so long?

**Review by dr-bonez — APPROVED** [review]

File: configurator/src/main.rs (hunk 1 above)
Comment (Dominion5254) [reply configurator/src/main.rs:225]: The port of C-Lightning-REST did not change, but clnrest (which RTL now exclusively uses as of 0.15.0) is listening on port 3010.

File: docker_entrypoint.sh (hunk 2 above)
Comment (Dominion5254) [reply docker_entrypoint.sh:43]: 30 seconds may be a bit on the long side, but this message seemed it was spamming the logs at a 1 second interval. Maybe 10-15 seconds would be more appropriate?

**Pull request merged**